lib/active_record/acts_as/instance_methods.rb

```diff
@@ -19,8 +19,8 @@ def changed?
 
       def actable_must_be_valid
         unless acting_as.valid?
-          acting_as.errors.each do |attribute, message|
-            errors.add(attribute, message) unless errors[attribute].include?(message)
+          acting_as.errors.each do |error|
+            errors.add(error.attribute, error.message) unless errors[error.attribute].include?(error.message)
```

Comment:
This change is only compatible with rails 6.1 and above.

Reply:
No it works on rails 6.0 as well see [here](https://api.rubyonrails.org/v6.0.0/classes/ActiveModel/Errors.html)

My CI runs against 6.0, 6.1 and rails master

Reply:
Your CI is not running against 6.0.  If you look at the build log you can see the 6.0 build is running 6.1.0.

https://github.com/chaadow/active_record-acts_as/blob/master/gemfiles/rails_6.0.gemfile#L6 has a version constraint that is too permissive.  It should be "~> 6.0.0".

Running the specs against 6.0 gives:
```
Failures:

  1) ActiveRecord::Base model with #acts_as called instance errors when validates_actable is set to true combines supermodel and submodel errors
     Failure/Error: errors.add(error.attribute, error.message) unless errors[error.attribute].include?(error.message)

     NoMethodError:
       undefined method `attribute' for :name:Symbol
     # ./lib/active_record/acts_as/instance_methods.rb:23:in `block in actable_must_be_valid'
```

Reply:
you're right, I'll look into that

Reply:
fixed in [5.0.1](https://rubygems.org/gems/active_record-acts_as/versions/5.0.1)